modules/nw-install-config-gateway-api.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * networking/about-contour-operator.adoc
+
+[id="nw-install-config-gateway-api_{context}"]
+= Installing and configuring Contour for Gateway API
+
+The following guide provides instructions for using link:https://gateway-api.sigs.k8s.io/[Gateway API] with the Contour on {product-title}.
+
+.Prerequisites
+
+* You installed an {product-title} cluster and the `oc` command line.
+* You installed the Contour Operator.
+//* Log in as a user with cluster-admin permission.
+
+.Procedure
+
+. Install Contour configured for Gateway API and dependent resources:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/projectcontour/contour-operator/v1.16.0/examples/gateway/gateway.yaml
+----
++
+[NOTE]
+====
+Envoy is exposed using a LoadBalancer service. Replace `gateway.yaml` with `gateway-nodeport.yaml` to use a NodePort service instead.
+====
+
+. Verify that all pods in the `projectcontour` namespace are running:
```

Comment:
The Contour install suggests that this namespace is optional. Should you state something like  _Verify that all pods in the namespace where you installed Contour are running_ ?